frontend/public/module/k8s/index.ts

```diff
@@ -376,6 +376,12 @@ export type NodeKind = {
     unschedulable?: boolean;
   };
   status?: {
+    capacity?: {
+      cpu?: string;
```

Comment:
It would be more accurate to just have

```
[key: string]: string;
```

here since any key is allowed

Reply:
ack.